src/commands/apm_cmds/publish.js

```diff
@@ -74,6 +76,9 @@ exports.builder = function (yargs) {
     }).option('http-served-from', {
       description: 'Directory where your files is being served from e.g. ./dist',
       default: null,
+    }).option('version-bump', {
+      description: 'Specify version bump type for to be published app',
+      default: null
```

Comment:
We can use yargs [choices](http://yargs.js.org/docs/#api-choiceskey-choices) so only values in the VERSION_BUMP_TYPES array are valid.

Reply:
I actually just tested this and the error message is quite crappy and would also require us to add `null` to the `VERSION_BUMP_TYPES` array. We can leave it as it is
<img width="545" alt="2018-09-11 at 4 32 25 pm" src="https://user-images.githubusercontent.com/447328/45366973-6c5aec80-b5e0-11e8-98db-97af314cf405.png">